lib/src/io_client.dart

```diff
@@ -72,3 +73,16 @@ class IOClient extends BaseClient {
     _inner = null;
   }
 }
+
+Uri _responseUrl(Request request, HttpClientResponse response) {
```

Comment:
K you want it static right? Was just following along with what we we're doing with free functions in `message.dart`